### PR TITLE
[DR-1647] Remove souce labs from build process

### PR DESCRIPTION
**Pull request by cristiangiagante**
## Background
  
We need to remove souce labs integration from our build process, we detect that is not mandatory to run to use our tests with travis CI, instead, this is doing some conflicts with travis CI tool with bad results like false positive with failing results on our tests and long times to run tests.
  
## Changes done
  
Remove souce labs integration from our build process from protractor
  
## Pending to be done
  
N/A
  
## Notes
  
Saucelabs was removed, take into account that saucelabs is a dependency from protractor and can't be removed from yarn file.

### CODE REVIEW

**Comment by andresmoschini** [comment]
I am not paying attention to this PR, @cristiangiagante let me know if you need my review.

**Comment by cristiangiagante** [comment]
@andresmoschini I test in origin and upstream the branch and saucelabs is disabled.

File: src/protractor-travis-conf.js [hunk 1]
```diff
@@ -1,8 +1,6 @@
 var configBuilder = require('./protractor-conf-builder');
 
 var config = configBuilder.getDefaults();
-config.sauceUser = process.env.SAUCE_USERNAME;
-config.sauceKey = process.env.SAUCE_ACCESS_KEY;
 config.multiCapabilities = [
```
Comment (andresmoschini) [inline src/protractor-travis-conf.js]: I think that this configuration is related to SauceLabs.

**Comment by andresmoschini** [comment]
Did you find for other references to SouceLabs? maybe in README.md or other files?

**Comment by cristiangiagante** [comment]
@andresmoschini Yes andres, I was forgot the readme file. I delete the multicapabilities already.

**Comment by cristiangiagante** [comment]
I will merge this pr I don't found more references in the solution to saucelabs implementation. You are agree @andresmoschini ?

**Comment by andresmoschini** [comment]
> I will merge this pr I don't found more references in the solution to saucelabs implementation. You are agree @andresmoschini ?

And the readme?

**Comment by andresmoschini** [comment]
imagen

**Comment by cristiangiagante** [comment]
Good catch, I clean it but I forget to send the push <3

**Pull request merged**